nilearn/glm/tests/test_utils.py

```diff
@@ -38,6 +45,40 @@ def test_z_score():
     assert_array_almost_equal(z_score(np.float32(1.e-100)), norm.isf(1.e-300))
 
 
+def test_z_score_opposite_contrast():
+    # put data fetching here
+    subject_data = fetch_spm_auditory()
```

Comment:
this won't work because the internet connection is mocked. functions that test dataset fetchers configure the mock response fetchers receive and fill it with fake data, but here you're better off not using the datasets package at all

Reply:
Yes, that's where I get the error. Concretely, I get a `TypeError: 'NoneType' object is not subscriptable` when trying to access `spm_auditory_data['events']` in `fetch_spm_auditory` function (line 2615 of nilearn/datasets/func.py).